### ISSUE TITLE
🚀 Group Routing support continuous path

### ISSUE THREAD

**Issue by myussufz** [gofiber/fiber#1098]
**Is your feature request related to a problem?**
No

**Describe the solution you'd like**
Routing not able to do continue prefix from the group instead it add "/". Example, the fiber router will output as "/api/store/s" but we expected "/api/stores"
```go
  e := fiber.New()
  store := e.Group("/api/store")
  store.Get("s", func(c *fiber.Ctx) (err error) {
     return c.SendString("get stores")
  }) // /api/stores
```
By commenting out this line, it is working fine, but we not sure whether the framework design to not support it.
![image](https://user-images.githubusercontent.com/21376844/103303681-8cc54400-4a41-11eb-9ead-e1c26f497c9d.png)

**Comment by kiyonlin** [comment]
It's related with #1089 .  I rarely see a const route split into two parts. But we can discuss about this with @Fenny and @ReneWerner87 .

**Comment by myussufz** [comment]
> It's related with #1089 . I rarely see a const route split into two parts. But we can discuss about this with @Fenny and @ReneWerner87 .

We are migrating from the echo framework, that framework is supporting this feature.

**Comment by ReneWerner87** [comment]
Don't know if we should allow this, in our concept the different routes and groups are always separated with a slash.

You can use this instead and i think this is more readable and understandable
```go
  e := fiber.New()
  store := e.Group("/api")
  store.Get("/store", func(c *fiber.Ctx) (err error) {
     return c.SendString("get store")
  })
  store.Get("/stores", func(c *fiber.Ctx) (err error) {
     return c.SendString("get stores")
  })
  ```

**Comment by myussufz** [comment]
> Don't know if we should allow this, in our concept the different routes and groups are always separated with a slash.
> 
> You can use this instead and i think this is more readable and understandable
> 
> ```go
>   e := fiber.New()
>   store := e.Group("/api")
>   store.Get("/store", func(c *fiber.Ctx) (err error) {
>      return c.SendString("get store")
>   })
>   store.Get("/stores", func(c *fiber.Ctx) (err error) {
>      return c.SendString("get stores")
>   })
> ```

If like this, then it should be released as a major release, since it will affect people who already use the framework because we test with v2.3.0, it is working fine. Also is better if gives the option to the developer to choose whether want to do continuous path or slash.

**Comment by kiyonlin** [comment]
> If like this, then it should be released as a major release, since it will affect people who already use the framework because we test with v2.3.0, it is working fine. Also is better if gives the option to the developer to choose whether want to do continuous path or slash.

This makes sense. Thank you for your suggestions!

**Comment by ReneWerner87** [comment]
@Fenny what do you think about the topic ? 
is this a feature for the next version or do we want to leave it like this and continue to delimit the groups with slashes ?

**Comment by Fenny** [comment]
I remember we discussed this internally when we introduced the `Group` method, we had to make a choice between both. Like @ReneWerner87 mentioned, we went with the separated slash method because it made more sense.

I'm afraid we cannot change this behavior because it will introduce a breaking change. We are not planning to move to `v3` anytime soon so I'm closing this issue for now. 

https://github.com/gofiber/fiber/issues/1098#issuecomment-752430800 example is a more readable/maintainable solution in my opinion.

Thank you for your suggestion though, thanks for taking the time to explore Fiber!
